.github/workflows/cd_gestor_rutas_entrega.yml

```
@@ -0,0 +1,98 @@
+      - name: Definir TAG de imagen con hora UTC-5 (Colombia)
+        if: github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, 'CD')
+        id: vars
+        run: |
+          TZ="America/Bogota"
+          export TAG=$(TZ=$TZ date +'%Y%m%d-%H%M%S')-${GITHUB_SHA::7}
+          echo "TAG=$TAG" >> $GITHUB_ENV
+          echo "Tag generado: $TAG"
+
+      - name: Autenticacion a GCP
+        if: github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, 'CD')
+        uses: google-github-actions/auth@v1
```

actionlint (1.7.4)

the runner of "google-github-actions/auth@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)